examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/index.tsx

```diff
@@ -89,6 +113,9 @@ const RideOptions = () => {
         }}
       />
       <ChoosePaymentMethod
+        selected={ride?.paymentMethodId?.length
```

Comment:
why do u need .length?
I think usePayments.paymentMethods.includes(ride?.paymentMethodId) is enough